streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -801,7 +801,7 @@ int numBuffered() {
      * @throws TaskMigratedException if the task producer got fenced (EOS only)
      */
     public boolean maybePunctuateStreamTime() {
-        final long timestamp = partitionGroup.timestamp();
+        final long timestamp = partitionGroup.streamTime();
```

Comment:
rename `timestamp` to `streamTime` (or `taskTime`) ?

Reply:
? It is renamed to streamTime?

Reply:
Sorry, duh, you meant the LHS. Ack